<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qdTest</class>
 <widget class="QDialog" name="qdTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Application</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="hblConnection">
     <item>
      <widget class="QGroupBox" name="gpbConnection">
       <property name="title">
        <string>Connection string</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="margin">
         <number>7</number>
        </property>
        <item>
         <widget class="QComboBox" name="cmbProvider"/>
        </item>
        <item>
         <widget class="QLineEdit" name="leConnection">
          <property name="text">
           <string>:memory:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbCheck">
          <property name="text">
           <string>Check</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Rows per page</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="margin">
         <number>7</number>
        </property>
        <item>
         <widget class="QLineEdit" name="leRowsPerPage">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="inputMask">
           <string>999999;0</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="hblQuery">
       <item>
        <widget class="QGroupBox" name="gpbQuery">
         <property name="title">
          <string>Query string</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="margin">
           <number>7</number>
          </property>
          <item>
           <widget class="QTextEdit" name="teQuery">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbExecute">
            <property name="toolTip">
             <string>Ctrl+Enter</string>
            </property>
            <property name="text">
             <string>Execute</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>teQuery</zorder>
         <zorder>pbExecute</zorder>
         <zorder>gpbResults</zorder>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="hblResults">
       <item>
        <widget class="QGroupBox" name="gpbResults">
         <property name="title">
          <string>Results</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="margin">
           <number>7</number>
          </property>
          <item>
           <widget class="QTableView" name="tbvResults">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hblForthBack">
            <item>
             <widget class="QPushButton" name="pbBack">
              <property name="text">
               <string>&lt; Back</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblPage">
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Current page:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frmPage">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>28</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="margin">
                <number>2</number>
               </property>
               <item>
                <widget class="QLabel" name="lblCurrentPage">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbForth">
              <property name="text">
               <string>Forth &gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hblClose">
     <item>
      <widget class="QLabel" name="lblUpdate">
       <property name="text">
        <string>Results updated:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frmUpdate">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblUpdateTime">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
